studio/lib/queries/admin.ts

```diff
@@ -46,9 +46,13 @@ export const EVENT_POSTINGS_QUERY = groq`
       _key, 
       "eventTitle": ${translatedFieldFragment("eventTitle")},
       "eventDescription": ${translatedFieldFragment("eventDescription")},
-      "locations": locations[],
+      "locations": locations[]{
+        "locationObject": ${translatedFieldFragment("locationObject")}
+      },
```

Comment:
Jeg tror _egentlig_ dette kan gjøres på en enklere måte og flate ut listen jeg altså. Aner ikke syntaksen men kan være verdt å sjekke ut på et tidspunkt for å forenkle.

Hypotetisk syntaks ([det finnes noe greier](https://www.sanity.io/answers/querying-nested-documents-with-groq-and-implementing-search-in-sanity-io))

```suggestion
      "locations": locations[]->${translatedFieldFragment("locationObject")}
      },
```

Reply:
Tester det!